### DISCUSSION TITLE
Font Weight?

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#2817] [General] [answered]
I don't see anything in the docs about setting the font weight. Light, Bold, Medium, Black, etc.

How do I set that and what are the value options?

**Comment by chr-ge** [comment, marked as answer]
These are the `fontWeights` in the [default theme](https://chakra-ui.com/docs/theming/theme#typography): 
```json
fontWeights: {
  normal: 400,
  medium: 500,
  bold: 700,
},
```
You can edit or add to these as you wish.
